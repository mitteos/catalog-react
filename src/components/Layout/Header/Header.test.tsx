import {screen} from "@testing-library/react";
import {renderWithRouter} from "tests/helpers/renderWithRouter";

describe("Header components",  () => {
    test("Header logo", () => {
        renderWithRouter(null)
        const logoElem = screen.findByAltText("logo")
        expect(logoElem).toMatchSnapshot()
    })
})
