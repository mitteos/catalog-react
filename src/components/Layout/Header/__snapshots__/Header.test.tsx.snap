// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header components Header logo 1`] = `Promise {}`;
